### ISSUE TITLE
GNU Parallel --will-cite

### ISSUE THREAD

**Issue by Flat** [mmstick/parallel#47]
Related to #39 `--will-cite` [1] is a GNU Parallel flag that also disables GNU Parallel's BibTeX citation output. Some scripts [2] use this flag, other than Parallel Rust failing to parse this command I believe Parallel Rust would work fine as a replacement.

1. https://www.gnu.org/software/parallel/man.html
2. https://github.com/AladW/aurutils/commit/683f1ea246f07835e38930af8471c9bef11a3fb9

**Comment by Flat** [comment]
It also appears this script uses the `-X` flag for GNU Parallel, which is described as

```
Multiple arguments with context replace. Insert as many arguments as the command line length permits. If multiple jobs are being run in parallel: distribute the arguments evenly among the jobs. Use -j1 to avoid this.

If {} is not used the arguments will be appended to the line. If {} is used as part of a word (like pic{}.jpg) then the whole word will be repeated. If {} is used multiple times each {} will be replaced with the arguments.

Normally -X will do the right thing, whereas -m can give unexpected results if {} is used as part of a word.

Support for -X with --sshlogin is limited and may fail.

See also -m.
```
I'm unsure if this should be another issue or if this is even desired behavior for Parallel Rust

**Comment by mmstick** [comment]
This is the default behavior of this implementation of the Parallel program. I should probably just change the argument parser to accept all arguments and instead give warnings for unsupported arguments, but continue on.

**Comment by Flat** [comment]
That's what I thought but the way it's worded in the GNU Parallel manual was a little confusing. Accepting invalid args and continuing might help to make Parallel Rust a drop-in replacement for GNU Parallel, but I can see where that may cause issues due to typos and such when running, I think.
